apps/condo/domains/marketplace/schema/InvoiceContext.js

```diff
@@ -80,25 +92,62 @@ const InvoiceContext = new GQLListSchema('InvoiceContext', {
             schemaDoc: 'How much money do we charge from companies. The percent.',
             isRequired: true,
             type: 'Decimal',
+            access: {
+                read: true,
+                create: userIsAdminOrIsSupport,
+                update: userIsAdminOrIsSupport,
+            },
+            hooks: {
+                resolveInput: ({ resolvedData, fieldPath }) => {
+                    return get(resolvedData, fieldPath, DEFAULT_IMPLICIT_FEE_PERCENT)
+                },
+            },
+        },
+
+        taxRegime: {
+            schemaDoc: 'The regime of counting taxes for company',
+            isRequired: true,
+            type: 'Select',
+            dataType: 'string',
+            options: TAX_REGIMES,
         },
 
-        vat: {
+        vatPercent: {
```

Comment:
Please add example value in docs. It’s not clear it it should like 0.01 or 1 or 1.01 or 1,01 for 1%

Reply:
For now, this field is limited to the options list (0, 10, 20).